packages/admin/README.md

```diff
@@ -40,6 +40,43 @@ await client.createProposal({
 });
 ```
 
+You can also optionally set the `simulate` flag as part of the `createProposal` request (as long as this is not a batch proposal) to simulate the proposal within the same request. You can override simulation parameters by setting the `overrideSimulationOpts` property, which is a `SimulationRequest` object.
+
+```js
+const proposalWithSimulation = await client.createProposal({
+  contract: {
+    address: '0xA91382E82fB676d4c935E601305E5253b3829dCD',
+    network: 'mainnet',
+    // provide this OR overrideSimulationOpts.transactionData.data
+    abi: JSON.stringify(contractABI),
+  },
+  title: 'Flash',
+  description: 'Call the Flash() function',
+  type: 'custom',
+  metadata: {
+    sendTo: '0xA91382E82fB676d4c935E601305E5253b3829dCD',
+    sendValue: '10000000000000000',
+    sendCurrency: {
+      name: 'Ethereum',
+      symbol: 'ETH',
+      decimals: 18,
+      type: 'native',
+    },
+  },
+  functionInterface: { name: 'flash', inputs: [] },
+  functionInputs: [],
+  via: '0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266',
+  viaType: 'EOA',
+  simulate: true, // set this to true
+  // optional
+  overrideSimulationOpts: {
+    transactionData: {
+      data: '0xd336c82d', // or instead of ABI, you can provide this
```

Comment:
Same here (since this is documentation, the clearer the better I think but just a nit comment)